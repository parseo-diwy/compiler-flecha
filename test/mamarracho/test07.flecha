
def foo = 42

def main =
  unsafePrintInt foo; unsafePrintChar '\n';
  let foo = 43 in
    unsafePrintInt foo; unsafePrintChar '\n';
  unsafePrintInt foo; unsafePrintChar '\n'

