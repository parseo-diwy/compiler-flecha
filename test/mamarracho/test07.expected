42
43
42
